app/js/views/services/default-service.jsx

```diff
@@ -0,0 +1,86 @@
+/* global React */
+
+import UserLogoutButton from 'js/views/user-logout-button';
+import FooterMenu from 'js/views/footer-menu';
+import TagList from 'js/views/tag-list';
+
+export default class Service extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: props.service.tags,
+      tags: []
+    };
+
+    this.foxbox = props.foxbox;
+    this.service = props.service;
+
+    this.onServiceStateChanged = this.onServiceStateChanged.bind(this);
+  }
+
+  componentDidMount() {
+    this.populateTags();
+
+    this.foxbox.addEventListener(
```

Comment:
I believe we have an issue in master, I didn't try to fix it in this PR. The issue is that we don't analyze service id in ```service-state-change``` event handler, so that if service1 changed state, then service2 that has this listener will override its state by the state of service2.